docs/index.rst

```diff
@@ -10,6 +10,8 @@
    Old 1.2.1 documentation is still available
    `here <https://code.google.com/p/psutil/wiki/Documentation>`__.
 
+.. contents::
```

Comment:
I don't like this because the table of contents is already visible in the left column